ExeterTeam25/group-software-project
# Image obfuscation

Obfuscating image names w hashing and a bug fix with profile picture updates.

To test:
- pip install new requirements
- Launch server
- Set profile picture on user
- Ensure filename is somewhat random
- set new profile picture
- Ensure old file is replaced with new one
- Verify no errors when changing name
- Submit image response to challenge
- Verify the name is fairly random looking
